flows/definition/flow.go

```diff
@@ -100,10 +100,13 @@ func (f *flow) validate() error {
 
 // Inspect enumerates dependencies, results etc
 func (f *flow) Inspect() *flows.FlowInfo {
+	assetRefs, parentRefs := f.extractAssetAndParentRefs()
```

Comment:
reworked this so that we don't have to go enumerating and parsing templates twice